### ISSUE TITLE
Improve the Decipherability of E2ETest Output

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#5827]
When E2ETest Fails, it's hard to parse out where something is going wrong, since the output shows every command sent across the process, so we see a lot of this:

![image](https://user-images.githubusercontent.com/835219/91125488-af3fee80-e656-11ea-8384-88a745fe54ec.png)

We should try to color code outputs, or omit bits that might be noisy. E.g our verbose Jest tests will show a single checkmark for each passing test, but will show detail if a test fails.

![image](https://user-images.githubusercontent.com/835219/91125585-ee6e3f80-e656-11ea-97f0-d52865f330be.png)

**Comment by NickGerleman** [comment]
Have a quick change to show colors, since it looks like our script does run Chalk and everything, but terminal detection was suppressing colors. 
![image](https://user-images.githubusercontent.com/835219/91130024-d4852a80-e65f-11ea-87b4-e953627019c1.png)

**Comment by asklar** [comment]
the ADO output will still show without colors. run_wdio does summarize the results at the bottom though.

**Comment by NickGerleman** [comment]
The above fix is to add colors to ADO. The change is just setting an ENV variable that chalk reads to tell it not do disable colors when it sees it's running in a non-interactive terminal. We have a similar trick in lerna builds commands. https://github.com/microsoft/react-native-windows/pull/5818/commits/7cd384f214b1474acd74bb78465a90055436ab53

**Comment by kmelmon** [comment]
let's use this issue to track making it more obvious when $ selector fails to find after timeout

**Comment by NickGerleman** [comment]
I ran into more issues, where tests seem to infinitely run without timing out, then the log was over 10MB and crashed Edge when I tried to view it.

There's the specific issues of various failures causing "undefined" or "null" messages, but I'm really hoping there's a way we can increase how actionable these logs are. I'm not sure I've ever seen one and had an idea of what went wrong, and I've had issues getting things running locally on multiple occasions.
